### DISCUSSION TITLE
Uncaught  ReferenceError: Cannot access '__WEBPACK_DEFAULT_EXPORT__' before initialization

### DISCUSSION THREAD

**Discussion by hiral-parmar-trivedi** [webpack/webpack#18671] [Q&A]
# Bug report
Getting Uncaught ReferenceError: Cannot access '__WEBPACK_DEFAULT_EXPORT__' before initialization in the latest CRA for redux flow
it's showing nested circular dependencies that I can't resolve as I have large amount of code.

I am converting my older react project to latest version with CRA via npm run ejected step.

CRA ejected folder have below version:
  "webpack": "^5.64.4",

**Other relevant information:**
Webpack version:  ^5.64.4
Node.js version: v20.16.0
Operating System: MAC OS

Kindly help me to prevent above error and it can allow circular dependency, Your help will be appreciated and helpful to so many devs.

**Comment by alexander-akait** [comment]
Sorry, it is a limitation, we can't handle all possible cases with circular dependencies, circular dependencies can be infinity, you need to change code to resolve it, if you provide reproducible test repo I will show you, also without code I can't say why it happends and can't debug

**Comment by hiral-parmar-trivedi** [comment]
@alexander-akait Is there any plugin or babel configuration available where older export syntax should also work?

**Reply by alexander-akait** [reply]
What do you mean?
